Added a section on alert channels reactions and checkup (#116)
reef-technologies/handbook: agreements.md

- It is recommendable for different developers to look at the subsequent iterations of the same code, as it may help catch bugs or look at the code differently.
- If a staff member realizes that everyone has refused to review a post, they should escalate the issue and discuss it with the team.

## Alerts channels

The various "alerts" channels receive issue messages from running deployments.

> In order to prevent alert blidness, each time new alert message appears it should be responded to. 

Reviewer: ```suggestion
In order to prevent alert blidness, each time new alert message appears it should be reacted to, i.e. resolved, or silenced. 
```

> Each channel has assigned people resposible for checking alerts appearing in them at least once per their workday.

Reviewer: ```suggestion
Each channel has assigned people (in the monkey registry) resposible for checking alerts appearing in them at least once per their workday. 
```

- It is recommendable for different developers to look at the subsequent iterations of the same code, as it may help catch bugs or look at the code differently.
- If a staff member realizes that everyone has refused to review a post, they should escalate the issue and discuss it with the team.

## Alerts channels

The various "alerts" channels receive issue messages from running deployments. Each channel has an assignee who is responsible for monitoring messages that arrive on that channel.

Similarly to the code review channel, the person responsible for a channel must properly annotate messages that indicate an issue using a proper reaction:
- 👀 (`:eyes:`) – “I am looking at this right now”,
- ✅ (`:white_check_mark:`) – “Issue resolved”,

> - 💤 (`:zzz:`) - “This doesn't require attention” / “This is already reported and will be addressed later”

Reviewer 2: IMO superflous, I would avoid developing our own pictographic language - we already have plenty of emoji reactions in use
IMO just leaving eyes emoji or even marking as resolved (even if resolution in this case meant "created a backlog item and thats good enough").

For a series of messages that arrive around the same time and are clearly related, it is enough to only react to the last message in the series.

In order to allow us to take the alerts more seriously and make them less likely to be ignored, we must keep the alerts channels free from noise. Ideally, all messages that arrive on alerts channels should be actionable.

**Staff members are encouraged to periodically skim through the alerts channels and do a quick checkup** on their state. It should be very easy to spot channels that are neglected:
- No reactions at all
- Lots of 💤 (issues being pushed back, too much noise)
- 👀, but no resolution

> Discussions on specific issues may take place in a thread related to the discussed issue.

Reviewer 2: ```suggestion
The various "alerts" channels receive issue messages from running deployments.
In order to prevent alert blidness, each time new alert message appears it should be responded to. 
Each channel has assigned people resposible for checking alerts appearing in them at least once per their workday.

Similarly to the code review channel, when reviewing an alert please following emoji reactions:
- 👀 (`:eyes:`) – “I am looking at this right now”,
- ✅ (`:white_check_mark:`) – “Issue resolved”,

Staff members should to periodically skim through the alerts channels and do a quick checkup on their state.
It should be easy to spot channels that are neglected, i.e. without any reactions or just 👀, but no resolutions

Reviewer 2: 1) sembr
2) made it shorter, since concise can be better as the more text we have in readme the less realistic is to expect everyone read it and remembers every detail (especially if some are much less important than others)
3) no need to explain how slack to write normal messages/use threads - if someone doesn't follow that guideline it doesn't really harm us. Either way it will be easy to see at glance if channel was neglected or not.
